tests/test_layout_objects.py

```diff
@@ -210,6 +210,10 @@ def test_inline_radios(self):
         html = render_crispy_form(test_form)
         assert html.count('radio-inline"') == 2
 
+    @pytest.mark.xfail(
```

Comment:
I marked this test as xfail because we run tests with crispy-forms version 1.x too.
I think about updating the test matrix in a separate pull request.